lib/src/json_schema/json_schema.dart

```diff
@@ -980,10 +980,9 @@ class JsonSchema {
       // Added or changed in draft2019_09: Core Vocabulary
       r'$anchor': (JsonSchema s, dynamic v) => null, // TODO: implement
       r'$defs': (JsonSchema s, dynamic v) => s._setDefs(v),
-      r'$id': (JsonSchema s, dynamic v) => null, // TODO: change behavior
+      // r'$id': (JsonSchema s, dynamic v) => null, // TODO: change behavior
```

Comment:
I guess this uses the old (not-yet-updated) `$id` behavior in order to get things working for you, correct? If so, :+1:

Reply:
That is correct.